site/en/docs/user-manual.md

```diff
@@ -1757,7 +1757,7 @@ a user-friendly way.
 
 ### Options for `bazel run` {:#bazel-run-options}
 
-#### `--run_under={{ "<var>" }}command-prefix{{ "</var>" }}` {:#run-under}
+#### `--run_under={{ "<var>" }}command-prefix{{ "</var>" }}` {:#run-run-under}
```

Comment:
nit: maybe `test-run-under` is a better name?

Reply:
This is the anchor for `bazel run --run_under`, perhaps you meant that the anchor for `bazel test --run_under` should be renamed as well?

Reply:
Oh, I see, yes, that would also be better.

Reply:
Done in a second commit. Also discovered a underscore versus minus mismatch in labels, good catch!